rust-toolchain

```diff
@@ -1 +1 @@
-1.61.0
+1.65.0
```

Comment:
if we are testing the msrv why not delete this file and let us always develop against the latest version?  The ci.yml could then also use `stable` instead of having to code this version in.

Reply:
huh, interesting.  which distro do you use that it gives you a good enough rust setup?